### DISCUSSION TITLE
How to use the Pro license in CI environment?

### DISCUSSION THREAD

**Discussion by evaddmx** [dashingsoft/pyarmor#1992] [Q&A] [answered]
It says the following in the docs
```
5. Fair use

If you have purchased Pyarmor License for one product, but you have aother product, and the total revenue of the other project is less than 100 x Pyarmor License fee, you could rent this license in your aother product


Pyarmor Pro License still could be used in CI/CD pipeline for low volume (100 runs per month)

```

But in the docs about registration it says:

```
Pyarmor Pro and Pyarmor Group  License can’t be used in CI/CD pipeline directly, but there is one workaround
```
Well, the workaround is using pyarmor locally, but in the Fair Use section says it can be used in CI/CD for low volume

How to use the Pro License for low volume in CI? Should I execute `pyarmor reg` every run?

**Comment by jondy** [comment, marked as answer]
Execute `pyarmor reg` every run, no more than 100 runs in 30 days, tha'ts fair use
